### ISSUE TITLE
1.16.4 Timed messages don't display?

### ISSUE THREAD

**Issue by bobhenl** [kangarko/ChatControl#441]
Hi, I don't know why, but I as administrator with * and nobody else can't see timed messages, there's probably something wrong but I don't know what. Will send you my debug.
[debug.zip](https://github.com/kangarko/ChatControl-Red/files/5756110/debug.zip)

**Comment by ElBananaa** [comment]
Hey!

Make sure your users have the permission `chatcontrol.chat.read`.
If they actually have this permission, since you are using luckperms, you coul try to use their verbose feature to see if there's no other missing permission.
Could you provide your server log and proxy logs too please?

**Comment by bobhenl** [comment]
Yea they have, and it's default permission. I'm setting perms right now and I think permissions are correct

**Comment by bobhenl** [comment]
Permissions:
![image](https://user-images.githubusercontent.com/18662086/103389249-03c80e80-4b0e-11eb-9756-ade11a03fa54.png)
[2020-12-31-5.log.gz](https://github.com/kangarko/ChatControl-Red/files/5756179/2020-12-31-5.log.gz) #lobby
[latest.log](https://github.com/kangarko/ChatControl-Red/files/5756180/latest.log) #bungee

**Comment by ElBananaa** [comment]
Thanks.
From what I can see, you forgot to configure a server-name in the server.properties, make sure to fix this issue.
Also make sure your plugins are all up-to-date.
If the issue persists, you'll have to wait for Kangarko to take a look at it.

**Comment by bobhenl** [comment]
I have set server-name=Lobby in config and yes plugins are up to date.

**Comment by kangarko** [comment]
Also ensure to give players chatcontrol.receive.announcer permission.

**Comment by kangarko** [comment]
Next version will show more Tips, make sure you have Show_Tips on true in settings.yml and read the messages to see why the timed message not shows. Comment below if you will still have this problem.

**Comment by bobhenl** [comment]
yes, I have tips and receive announcer too, so where's the problem?

**Comment by ElBananaa** [comment]
> yes, I have tips and receive announcer too, so where's the problem?

As he said, please update to the latest version, your issue should be explained in the logs.
If you can't find the issue, please provide us your `/chc debug` file, and your latest.log file

**Comment by bobhenl** [comment]
[debug.zip](https://github.com/kangarko/ChatControl-Red/files/5757904/debug.zip)
[latest.log](https://github.com/kangarko/ChatControl-Red/files/5757925/latest.log)

**Comment by ElBananaa** [comment]
Thanks.
I don't see what's wrong with your actual messages.
Since you can see the messages with the `*` permission, but players can, I assume this is somehow related to your permissions.
Use luckperms' verbose feature in order to know which permission(s) is/are missing.
`/lp verbose on chatcontrol`

**Comment by bobhenl** [comment]
No, I don't see it with * neither.

**Comment by kangarko** [comment]
@bobhenl do you see any tips-related messages in the console when the announcement should be fired, but it is not shown?

**Comment by bobhenl** [comment]
No it isn't neither in console, check my file please https://pastebin.com/brth1Pii Only triggered is the first one - about winter, nothing else and console doesn't show neither any of these (searched for their prefix)

**Comment by kangarko** [comment]
Hey, I got it to work, all I did was I put a delay into the winter-related message about turning off snow and then they were iterated properly.

<img width="1417" alt="Screenshot 2021-01-07 at 18 16 31" src="https://user-images.githubusercontent.com/1918426/103922756-7e97b780-5114-11eb-95d8-5ca6483e2249.png">

The only case it doesn't send is when the sending server is empty. We cannot send packets from an empty server.
